### DISCUSSION TITLE
Hypno feels like dronification

### DISCUSSION THREAD

**Discussion by dDeepLb** [littlesera/LSCG#383] [Ideas]
You just do what you been said. 
Only actions.
You can't talk.
Can't hear anything except hypnotizer or when called by your name.

Boooring...

Add ability to give right of voice or setting if player can talk when hypnotized.
Same goes to hearing.
Works in RP kind of "Tell me... {something}".

**Comment by anchillabc** [comment]
The below represents a comment I got regarding hypno without asking about it, or mentioning this discussion; I'm adding it without my own comments, which I may formulate later:
Sasaki Izumi: It's sad that you can't talk during hypnosis
Sasaki Izumi: Kinda killing the rp

**Comment by littlesera** [comment]
Yeah, this is becoming a higher-priority item.

**Comment by littlesera** [comment]
Addressed in #396 
* Added "allow speech" and "silence" trigger configurations so users can set custom phrases that will allow/block their speech while hypnotized. 
* Configured similar to regular hypno triggers/awakeners.
* Speech is always blocked at start of hypnosis, and no default allow/block triggers are set on lscg update
